docs/running-node.md

```diff
@@ -50,7 +50,7 @@ and contains the genesis and persistent_peers files.
 
 2. Configure dclcli:
     * `dclcli config chain-id <chain-id>`
-      * Use `dcl-test-net-chain` if you want to connect to the persistent Test Net
+      * Use `testnet` if you want to connect to the persistent Test Net
```

Comment:
Please change the proposed test net name in `how-to.md` (search for `dcl-test-net-chain` in the project).

Reply:
Fixed